cmd/compose.go

```diff
@@ -110,67 +110,66 @@ Specify config location when executing the command:
 
 func compose() {
 	// Build plugin generator config
-	prepConfig()
+	configErr := prepConfig()
+	if configErr != nil {
+		LogWithCommand.Fatalf("failed to prepare config: %s", configErr.Error())
+	}
 
 	// Generate code to build the plugin according to the config file
 	LogWithCommand.Info("generating plugin")
-	generator, err := p2.NewGenerator(genConfig, databaseConfig)
-	if err != nil {
-		LogWithCommand.Debug("initializing plugin generator failed")
-		LogWithCommand.Fatal(err)
+	generator, constructorErr := p2.NewGenerator(genConfig, databaseConfig)
+	if constructorErr != nil {
+		LogWithCommand.Fatalf("initializing plugin generator failed: %s", constructorErr.Error())
 	}
-	err = generator.GenerateExporterPlugin()
-	if err != nil {
-		LogWithCommand.Debug("generating plugin failed")
-		LogWithCommand.Fatal(err)
+	generateErr := generator.GenerateExporterPlugin()
+	if generateErr != nil {
+		LogWithCommand.Fatalf("generating plugin failed: %s", generateErr.Error())
 	}
 	// TODO: Embed versioning info in the .so files so we know which version of vulcanizedb to run them with
-	_, pluginPath, err := genConfig.GetPluginPaths()
-	if err != nil {
-		LogWithCommand.Debug("getting plugin path failed")
-		LogWithCommand.Fatal(err)
+	_, pluginPath, pathErr := genConfig.GetPluginPaths()
+	if pathErr != nil {
+		LogWithCommand.Fatalf("getting plugin path failed: %s", pathErr.Error())
 	}
-	fmt.Printf("Composed plugin %s", pluginPath)
 	LogWithCommand.Info("plugin .so file output to ", pluginPath)
 }
 
 func init() {
 	rootCmd.AddCommand(composeCmd)
 }
 
-func prepConfig() {
+func prepConfig() error {
```

Comment:
This is probably outside the scope of this PR, but I wonder if it would make sense to move `prepConfig` into `root.go` or another higher level command file. I think it feels a little weird that this method definition is in `compose.go` when `execute.go` is also using it.